model/src/main/kotlin/org/digma/intellij/plugin/model/discovery/MethodInfo.kt

```diff
@@ -21,10 +21,14 @@ data class MethodInfo(
         }
     }
 
-    fun getRelatedCodeObjectIds(): List<String> {
+    fun getRelatedCodeObjectIdsWithType(): List<String> {
         return spans.stream().map(SpanInfo::idWithType).collect(Collectors.toList())
     }
 
+    fun getRelatedCodeObjectIds(): List<String> {
+        return spans.stream().map(SpanInfo::id).collect(Collectors.toList())
+    }
+
```

Comment:
I did global Intellij refactoring via Shift + F6, I renamed existing getRelatedCodeObjectIds to getRelatedCodeObjectIdsWithType (because this is exactly what that method does ) + I created new needed for me getRelatedCodeObjectIds that works only with id instead of taking into account idWithType.
<img width="935" alt="image" src="https://user-images.githubusercontent.com/117861250/209654237-42b1df0c-ced0-4ed7-b488-3f6d7960e65b.png">

So for sure all places that used before getRelatedCodeObjectIdsWithType now are using the same implementation that they used before